timescaledb/overview/release-notes/changes-in-timescaledb-2.md

```diff
@@ -72,9 +72,9 @@ to customize behaviors when needed.**
 For example, in TimescaleDB 2.0 we've separated the automation of _refreshing_ continuous aggregate
 data from the core functionality, giving users the option to manually refresh the data and, if
 desired, add automation via a policy. This also makes this feature consistent with TimescaleDB's
-other policy-driven features – such as retention, reordering, and compression – which offer both
+other policy-driven features—such as retention, reordering, and compression—which offer both
 manual control and automation via policies. We have also opened up our jobs scheduling framework
-to enable user-defined actions, i.e, custom background jobs that users can define themselves.
+to enable user-defined actions (custom background jobs that users can define themselves).
```

Comment:
```suggestion
to enable user-defined actions. User-defined actions are custom background jobs that you can define yourself.
```

Try to avoid parenths if you can, and also make sure we're not referring to third-person users.